plugins/Installation/lang/en.json

```diff
@@ -131,7 +131,7 @@
         "SystemCheckWriteDirs": "Directories with write access",
         "SystemCheckWriteDirsHelp": "To fix this error on your GNU\/Linux system, try typing in the following command(s)",
         "SystemCheckZlibHelp": "You need to configure and rebuild PHP with \"zlib\" support enabled, --with-zlib.",
-        "SystemCheckCronArchiveProcess": "Archive Cron",
+        "SystemCheckCronArchiveProcess": "Setup Cron",
```

Comment:
I'm still finding it useful if it could be a bit more clear what the outcome/result is of fixing this warning so users will pay more attention to it as "Setup cron" you would likely simply ignore. Like "Setup Cron (faster report loading)" or something. Not too important but I think it will get the attention of users a lot more